spec/models/restaurant_spec.rb

```diff
@@ -1,15 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe Restaurant, type: :model do
+  #before(:each) do
```

Comment:
You probably want to remove these commented lines

Reply:
fixed